script.js

```diff
@@ -1,4 +1,116 @@
+//-------------------------------------------Base Code---------------------------------------
+//array to store final number concatenated by the players
+var diceRolls = [];
+//random number rolled by dice 1
+var dice1 = 0;
+//random number rolled by dice 2
+var dice2 = 0;
+//number of turns completed. Each round there will be 2 turns since there are only 2 players
```

Comment:
Very helpful comments. Try to have clearer variable names too so that comments become less necessary.